src/main/java/com/sigopt/model/ConvergenceCriteria.java

```diff
@@ -0,0 +1,55 @@
+package com.sigopt.model;
+
+public class ConvergenceCriteria extends StructObject {
```

Comment:
this looks unused

Reply:
Can be used in this way:

        Experiment experiment = Experiment.create(
            new Experiment.Builder()
                .name("Franke Optimization (Java)")
                ....
                .trainingMonitor(
                    new TrainingMonitor.Builder()
                        .maxCheckpoints(10)
                        .earlyStoppingCriteria(Arrays.asList(
                            new ConvergenceCriteria.Builder()
                                .name("test")
                                .metric("mootrack")
                                .lookbackCheckpoints(3)
                                .minCheckpoints(4)
                                .build()
                        ))
                        .build()
                )
                .build())
            .call();